requirements.txt

```diff
@@ -20,3 +20,5 @@ argparse>=1.3.0; python_version < '2.7'
 logutils>=0.3.3; python_version < '2.7'
 importlib>=1.0.3; python_version < '2.7'
 unittest2>=1.0.0; python_version < '2.7'
+# Aexpect is now used on Docker plugin
+aexpect>=1.2.0
```

Comment:
Yep, I'm sorry I forgot to add it... Hopefully we'll remove it once we separate the plugins...